frontend/apps/filemanager/filemanagersearch.lua

```diff
@@ -206,8 +206,8 @@ function Search:ShowSearch()
                     },
                 },
             },
-            width = Screen:getWidth() * 0.8,
-            height = Screen:getHeight() * 0.2,
+            width = math.floor(Screen:getWidth() * 0.8),
```

Comment:
As an aside, I'm not sure if there's much rationale behind this one being 0.8 and many of those others being 0.95 or whatever.

Reply:
True ;o).

That was vaguely what I was hoping to make clear bu using "constants" from somewhere, but as the intermediate commit explain, doing that while handling dimension changes & rotation would require greater change (e.g., moving that to a table in Screen (and not ui/size), which would be responsible for keeping it updated, and then accessing it via a getter to never use a local cached value).